### PR TITLE
Impartner community pack context fix

### PR DESCRIPTION
**Pull request by edik24**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by Shellyber — CHANGES_REQUESTED** [review]
@edik24 
You'll need to update the yml file, readme file, add release notes and bump the pack version as well.

**Comment by edik24** [comment]
> @edik24 You'll need to update the yml file, readme file, add release notes and bump the pack version as well.

actually the yml and readme are ok, the problem was with the context I prepare.
added release notes and updated version

File: Packs/Impartner/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Impartner
+
+- changed path for context argument "assigned PST engineer"
```
Comment (Shellyber) [inline Packs/Impartner/ReleaseNotes/1_0_1.md:6]:
```suggestion
- Fixed an issue where the context path of *assigned PST engineer* was incorrect in **impartner-get-account-id** command.
```

**Review by Shellyber — APPROVED** [review]
nice

**Pull request merged**